src/components/PopupWithImage.js

```diff
@@ -7,9 +7,9 @@ export default class PopUpWithImage extends Popup {
 
     super(popupSelector);
 
-    /* this._name = name;
+    this._name = name;
 
-    this._link = link; */
+    this._link = link;
 
```

Comment:
```js

  close() {

    super.close();

  }
```

You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.